queue_services/entity-emailer/src/entity_emailer/email_templates/common/style.html

```diff
@@ -74,7 +74,7 @@
 
 .title-message {
   font-size: 24px;
-  line-height: 34px;
+  padding-bottom: 16px;
```

Comment:
The previous CSS worked fine for single-line titles but looked terrible for multi-line titles such as the current email, so I changed it. See screenshots.